RELEASE_NOTES.md

```diff
@@ -1,20 +1,31 @@
-# Cosmos SDK v0.39.1 Release Notes
+# Cosmos SDK v0.39.2 Release Notes
 
-This release fixes the [issue affecting the accounts migration](https://github.com/cosmos/cosmos-sdk/issues/6828) from v0.38 to v0.39.
+This release fixes various bugs and brings coin's denom validation to the *Cosmos SDK 0.39* release series.
 
-See the [Cosmos SDK 0.39.1 milestone](https://github.com/cosmos/cosmos-sdk/milestone/29?closed=1) on our issue tracker for details.
+See the [Cosmos SDK 0.39.2 milestone](https://github.com/cosmos/cosmos-sdk/milestone/30?closed=1) on our issue tracker for details.
 
-## Remove custom JSON serialization for account types
+## Allow ValidateDenom() to be customised per application
 
-Account types JSON serialization has now changed to Amino. Changes are significant (e.g. integers are treated
```

Comment:
This is removed?

Reply:
Oh no, I've just erased the old RELEASE_NOTES.md and written a new one.

Reply:
I thought we were treating RELEASE_NOTES as a file specific for a release - unlike CHANGELOG.md, which instead is meant to collect changes introduced in all releases.

Reply:
Ahhh... that makes more sense. Looks good to me.